Feature/145/about content on mobile

GabeDiniz opened the pull request:
✨ **Issue:** https://github.com/LaurierHawkHacks/Landing/issues/145
🔀 **Branch:** feature/145/about-content-on-mobile

**✏ What was changed?**
- Update SVG for mobile
- Change text color to match Hero section (hopefully its more readable)

Let me know what you guys think about the text color. It matches the Hero section now and I think its a little more readable. I don't know how I feel about the About Title and Content being different colors though. 

See the last screenshot to see the possible alternative for text-color. 


**📜 Screenshots**
Width: 390px
![image](https://github.com/LaurierHawkHacks/Landing/assets/55710093/693e5183-2b40-48ca-8384-bddd302ba7e1)

Width: 650px
![image](https://github.com/LaurierHawkHacks/Landing/assets/55710093/3a5d1871-3f7d-46bb-b638-5d7608ca6fa3)

Possible alternative?
![image](https://github.com/LaurierHawkHacks/Landing/assets/55710093/454b0f96-3a22-4e6c-ba31-56419cf01863)

GabeDiniz commented:
Looks better with the gradient:
![image](https://github.com/LaurierHawkHacks/Landing/assets/55710093/ec3f77a0-96e8-416a-bcf0-786882588986)
![image](https://github.com/LaurierHawkHacks/Landing/assets/55710093/c7d2f844-5428-4531-b4e4-f68ec6d11206)

teoh4770 commented on src/assets/hero-about/hero-about-desktop-bak.svg:
the naming of the image is weird? Can you change it?

...bak.svg?

teoh4770 reviewed (commented):
Got one question: Why do we have repeated images in the assets/hero-about?

![Screenshot 2024-03-24 at 6 12 34 PM](https://github.com/LaurierHawkHacks/Landing/assets/98545971/e0860b26-6e5a-4c72-840f-8cd82e802c98)

teoh4770 commented:
and I feel like **the about section paragraphs' color can be darker to make the text more obvious**.
I have run the color choices in the web accessibility color contrast checker, and it suggests that as well.

teoh4770 commented:
![Screenshot 2024-03-24 at 6 31 57 PM](https://github.com/LaurierHawkHacks/Landing/assets/98545971/f2cdcb5c-74c5-4efc-bf61-15b6f9875942)

I also find that the spacing of the content is weird in desktop view.
I know it is not under your scope of your pr, we can make an issue for that after we have addressed this one first.

teoh4770 commented:
but besides that, I think it is better!

GabeDiniz commented:
> Got one question: Why do we have repeated images in the assets/hero-about?
> 
> ![Screenshot 2024-03-24 at 6 12 34 PM](https://private-user-images.githubusercontent.com/98545971/316346648-e0860b26-6e5a-4c72-840f-8cd82e802c98.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzMjE2MjcsIm5iZiI6MTcxMTMyMTMyNywicGF0aCI6Ii85ODU0NTk3MS8zMTYzNDY2NDgtZTA4NjBiMjYtNmU1YS00YzcyLTg0MGYtOGNkODJlODAyYzk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI0VDIzMDIwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlZjViZjA5ZmQxY2NkMjcwOWU3MDZhNDVhZjg4MmRjMzJiNzA4OTI4NDMzMTI1M2E5ZDg3MDMyMjNjYmM3ZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aimZhvDRTJP_G-tLf2O-z-TkuqkbmT5jhMG4OVEqbUM)

I was gonna remove that :) just waiting to see if everyone was happy with the final design. I didn't want to get rid of it until we all came to an agreement on the styling. Sorry about that, I should've said said that 😅

GabeDiniz commented:
> and I feel like **the about section paragraphs' color can be darker to make the text more obvious**. I have run the color choices in the web accessibility color contrast checker, and it suggests that as well.

@teoh4770 do you have a color suggestion? What do you think of this (#1D4549)
![image](https://github.com/LaurierHawkHacks/Landing/assets/55710093/edf5f905-9989-451e-895c-ba6273e76891)

GabeDiniz commented:
> ![Screenshot 2024-03-24 at 6 31 57 PM](https://private-user-images.githubusercontent.com/98545971/316347733-f2cdcb5c-74c5-4efc-bf61-15b6f9875942.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzMjE2MjYsIm5iZiI6MTcxMTMyMTMyNiwicGF0aCI6Ii85ODU0NTk3MS8zMTYzNDc3MzMtZjJjZGNiNWMtNzRjNS00ZWZjLWJmNjEtMTViNmY5ODc1OTQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI0VDIzMDIwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMwN2RhZDZiYTQ3ZWM5NDFmODU0Yzk2YmY1ZGM3NmZiM2MwMmQyNzI4YTg1NGU3ZjdiNzg2MmRiMGU5MTk4NTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OR2QCZAztprYCziTT09ZRj0oLwMZ94Z49xP33fYqytE)
> 
> I also find that the spacing of the content is weird in desktop view. I know it is not under your scope of your pr, we can make an issue for that after we have addressed this one first.

You are right, its totally weird LOL.. I just fixed it. Should look a lot better now for all breakpoints.

teoh4770 commented:
> > and I feel like **the about section paragraphs' color can be darker to make the text more obvious**. I have run the color choices in the web accessibility color contrast checker, and it suggests that as well.
> 
> @teoh4770 do you have a color suggestion? What do you think of this (#1D4549)
> ![image](https://github.com/LaurierHawkHacks/Landing/assets/55710093/edf5f905-9989-451e-895c-ba6273e76891)
> 

As long as it's clear to see, and the colour is aligned with the title like the one you have. Others can suggest the colour as well

teoh4770 commented:
> > Got one question: Why do we have repeated images in the assets/hero-about?
> > 
> > ![Screenshot 2024-03-24 at 6 12 34 PM](https://private-user-images.githubusercontent.com/98545971/316346648-e0860b26-6e5a-4c72-840f-8cd82e802c98.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzMjE2MjcsIm5iZiI6MTcxMTMyMTMyNywicGF0aCI6Ii85ODU0NTk3MS8zMTYzNDY2NDgtZTA4NjBiMjYtNmU1YS00YzcyLTg0MGYtOGNkODJlODAyYzk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI0VDIzMDIwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlZjViZjA5ZmQxY2NkMjcwOWU3MDZhNDVhZjg4MmRjMzJiNzA4OTI4NDMzMTI1M2E5ZDg3MDMyMjNjYmM3ZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aimZhvDRTJP_G-tLf2O-z-TkuqkbmT5jhMG4OVEqbUM)
> 
> I was gonna remove that :) just waiting to see if everyone was happy with the final design. I didn't want to get rid of it until we all came to an agreement on the styling. Sorry about that, I should've said said that 😅
We can always revert back to the change if we want to get back the original pic👍
And no worries though

aidantrabs commented:
We'll talk about this tonight
